Sources/Sentry/SentryThreadInspector.m

```diff
@@ -28,27 +64,89 @@ - (id)initWithStacktraceBuilder:(SentryStacktraceBuilder *)stacktraceBuilder
 {
     NSMutableArray<SentryThread *> *threads = [NSMutableArray new];
 
-    SentryCrashMC_NEW_CONTEXT(context);
-    [self.machineContextWrapper fillContextForCurrentThread:context];
+    @synchronized(self) {
```

Comment:
`m`: Why do we need the lock here?

Reply:
We don't. Inheritance from the old code.